test/test_helper.exs

```diff
@@ -8,12 +8,15 @@ Application.put_all_env(
   honeybadger: [
     environment_name: :test,
     api_key: "abc123",
-    origin: "http://localhost:4444"
+    origin: "http://localhost:4444",
+    insights_enabled: true
   ]
 )
 
 ExUnit.start(assert_receive_timeout: 1000, refute_receive_timeout: 1000)
 
+Code.require_file("test/support/insights_case.ex")
```

Comment:
Usually, the support directory is configured to be included automatically and you don't have a manual require.

Reply:
Ah yeah, [this](https://github.com/honeybadger-io/honeybadger-elixir/pull/591/commits/def0c08fd8bcd31bba1dbc45d7398952de585e3f) is better.